### PR TITLE
conflict teset

### PR DESCRIPTION
**Pull request by Ajinkya-Nawarkar**

### CODE REVIEW

File: frontend/logout.php [hunk 1]
```diff
@@ -8,7 +8,8 @@
         session_start();
 
         // destroy session
-        session_unset();
+        session_unst();
```
Comment (Ajinkya-Nawarkar) [inline frontend/logout.php]: ajsfbasfn

File: frontend/logout.php [hunk 2]
```diff
@@ -8,7 +8,8 @@
         session_start();
 
         // destroy session
-        session_unset();
```
Comment (JackBetbeze) [inline frontend/logout.php]: heyoo

Comment (aaronval47) [inline frontend/logout.php]: asdfnsdlfslj

Comment (ChandlerU) [inline frontend/logout.php]: hhhhh

Comment (Ajinkya-Nawarkar) [reply frontend/logout.php]: n jaf

**Pull request merged**